services/alertmanager/alertmanager.go

```diff
@@ -64,7 +64,11 @@ func (svc *Service) Run(ctx context.Context) {
 	svc.l.Info("Starting...")
 	defer svc.l.Info("Done.")
 
-	svc.createDataDir()
+	err := dir.CreateDataDir(alertmanagerDataDir, "pmm", "pmm", dirPerm)
```

Comment:
Before that, we should call dir.CreateDataDir("/srv/alertmanager")

Please also do the same for /srv/victoriametrics and /srv/victoriametrics/data

Reply:
> Please also do the same for /srv/victoriametrics and /srv/victoriametrics/data

added it to victoriametrics package, wasn't sure if it should stay in that one or the vmalert package, let me know if otherwise